### DISCUSSION TITLE
Projects beta: "Share feedback" links to non-existent Community category

### DISCUSSION THREAD

**Discussion by iandundas** [community/community#20419] [Projects and Issues] [answered]
For example [here](https://docs.github.com/en/issues/trying-out-the-new-projects-experience/about-projects):

<img width="400" alt="image" src="https://user-images.githubusercontent.com/1131967/178683304-d6234789-8747-466e-a372-4a06862ffbfa.png">

The link takes me to file feedback at _[https://github.com/github-community/community/discussions/categories/issues-feedback](https://github.com/github-community/community/discussions/categories/issues-feedback)_, however that category does not exist and you cannot create a discussion there.

**Comment by radar-blip** [comment, marked as answer]
Interesting. Looks like they may have changed the name of the URL and left the old one. 
You can report it as a bug here: https://support.github.com/contact/bug-report

**Reply by iandundas** [reply]
Will do 👍🏻

**Reply by iandundas** [reply]
I [filed a bug (private link for GitHub)](https://support.github.com/ticket/personal/0/1704636)

**Reply by radar-blip** [reply]
Ok good. Hope it gets fixed soon!
